assets/js/custom/MediaLib.js

```diff
@@ -35,19 +35,39 @@ export function MediaLib (packageName, mediaSearchPath, flavor, assetsDir,
       showTopBarDefault()
     }
 
+    // accepted flavors
+    const acceptedFlavors = [
+      'pocketcode'
+    ]
+    if ($.inArray(flavor, acceptedFlavors) === -1) {
```

Comment:
It would be great if we could get rid of jquery and its overhead.
you could use indexOf here

Reply:
Note: no need to change it now; we can remove jquery in a separate user story